Jellyfin.Api/Controllers/DashboardController.cs

```diff
@@ -45,9 +46,9 @@ public class DashboardController : BaseJellyfinApiController
     /// <response code="404">Server still loading.</response>
     /// <returns>An <see cref="IEnumerable{ConfigurationPageInfo}"/> with infos about the plugins.</returns>
     [HttpGet("web/ConfigurationPages")]
+    [Authorize(Policy = Policies.RequiresElevation)]
```

Comment:
Does the other en dpoint need to be public?

Reply:
Right now, yes. That endpoint is loaded without any authentication in an iframe by the web client. So changing that would need additional work.

Reply:
Should we add a note or something?

Reply:
Sure